en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+# Cocos Creator v2.1.0 Release Notes
+
+__Cocos Creator__ __v2.1__, adds support for 3D features such as: **3D model rendering**, **3D Camera**, **3D skeletal animation**, and **3D point selection**. The __Cocos Creator__ editor now natively supports parsing 3D model files in **FBX format**. It's that simple, no additional workflow is required!
+
+In the previous version, v2.0, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer to officially introduce 3D support. The addition of 3D features can greatly enrich the expressiveness of 2D games, reduce the resource overhead of 2D games, and bring more imagination and ideas to expand games.
```

Comment:
In the previous v2.0, Cocos Creator had used a Cocos3D based 3D renderer. With this foundation, it could officially introduce 3D support.

Reply:
I used: In the previous __v2.0__, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer. With this foundation, 3D support could officially be introduced.